.travis.yml

```diff
@@ -29,6 +29,8 @@ addons:
 
 install:
     - pip install -U pip tox codecov
+    - git clone https://github.com/Jekub/Wapiti /tmp/wapiti
+    - pushd /tmp/wapiti && make && sudo make install && popd
```

Comment:
👍 
heh, I wonder why does it work - we're using `sudo: false` on top of travis.yml